dipy/reconst/tests/test_ivim.py

```diff
@@ -185,7 +185,8 @@ def test_mask():
     Test whether setting incorrect mask raises and error
     """
     mask_correct = data_multi[..., 0] > 0.2
-    mask_not_correct = np.array([[False, True, False], [True, False]])
+    mask_not_correct = np.array([[False, True, False], [True, False]],
+                                dtype=object)
```

Comment:
Could it work if you use `dtype=np.bool` instead of `dtype=object`?

Reply:
I guess so, but since I am unable to reproduce the issue locally I am not be able to confirm. I do to not have a preference. Do you prefer `dtype=np.bool`?

Reply:
yes, I prefer it

Reply:
Changed in cae74f2.